examples/model_selection/plot_roc.py

```diff
@@ -121,28 +137,122 @@
 roc_auc["macro"] = auc(fpr["macro"], tpr["macro"])
 
 # Plot all ROC curves
-plt.figure()
-plt.plot(fpr["micro"], tpr["micro"],
-         label='micro-average ROC curve (area = {0:0.2f})'
-               ''.format(roc_auc["micro"]),
-         color='deeppink', linestyle=':', linewidth=4)
+fig, ax = plt.subplots()
+ax.plot(fpr["micro"], tpr["micro"],
+        label='micro-average ROC curve (area = {0:0.2f})'
+              ''.format(roc_auc["micro"]),
+        color='deeppink', linestyle=':', linewidth=4)
 
-plt.plot(fpr["macro"], tpr["macro"],
-         label='macro-average ROC curve (area = {0:0.2f})'
-               ''.format(roc_auc["macro"]),
-         color='navy', linestyle=':', linewidth=4)
+ax.plot(fpr["macro"], tpr["macro"],
+        label='macro-average ROC curve (area = {0:0.2f})'
+              ''.format(roc_auc["macro"]),
+        color='navy', linestyle=':', linewidth=4)
 
 colors = cycle(['aqua', 'darkorange', 'cornflowerblue'])
 for i, color in zip(range(n_classes), colors):
     plt.plot(fpr[i], tpr[i], color=color, lw=lw,
              label='ROC curve of class {0} (area = {1:0.2f})'
              ''.format(i, roc_auc[i]))
 
-plt.plot([0, 1], [0, 1], 'k--', lw=lw)
-plt.xlim([0.0, 1.0])
-plt.ylim([0.0, 1.05])
-plt.xlabel('False Positive Rate')
-plt.ylabel('True Positive Rate')
-plt.title('Some extension of Receiver operating characteristic to multi-class')
-plt.legend(loc="lower right")
-plt.show()
+ax.plot([0, 1], [0, 1], 'k--', lw=lw)
+ax.set_xlim([0.0, 1.0])
+ax.set_ylim([0.0, 1.05])
+ax.set_xlabel('False Positive Rate')
+ax.set_ylabel('True Positive Rate')
+ax.set_title('An extension of ROC to multi-class using One-vs-Rest')
+ax.legend(loc="lower right")
+fig.show()
+
+###############################################################################
+# One-vs-One
+# ---------------------
+# The One-vs-One scheme compares every unique pairwise combination of classes.
+#
+# Compute the AUC score
+# .....................
+# The ROC area can be approximated by taking the average either weighted
+# uniformly (macro) or by prevalence.
```

Comment:
Above we have a similar statement but use different terms, such as "support" above being "prevalence" here

Reply:
The language about prevalence and support is removed. (Less terminology reads better)